packages/admin/cms-admin/src/userPermissions/user/UserPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { gql, useQuery } from "@apollo/client";
+import { MainContent, Tab, Tabs, Toolbar, ToolbarBackButton, ToolbarTitleItem } from "@comet/admin";
+import { Box, CircularProgress } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { UserBasicData } from "./basicData/UserBasicData";
+import { ContentScopeGrid } from "./permissions/ContentScopeGrid";
+import { PermissionGrid } from "./permissions/PermissionGrid";
+import { GQLUserPageQuery, GQLUserPageQueryVariables } from "./UserPage.generated";
+
+export const UserPage: React.FC<{ userId: string }> = ({ userId }) => {
+    const { data, error, loading } = useQuery<GQLUserPageQuery, GQLUserPageQueryVariables>(
+        gql`
+            query UserPage($id: String!) {
+                user: userPermissionsUserById(id: $id) {
+                    name
+                    email
+                }
+            }
+        `,
+        {
+            variables: { id: userId },
+        },
+    );
+
+    if (error) {
+        throw new Error(error.message);
+    }
+
+    if (loading || !data) {
+        return <CircularProgress />;
+    }
+
+    return (
+        <>
+            <Toolbar>
+                <ToolbarBackButton />
+                <ToolbarTitleItem>
+                    <TitleText>{data.user.name}</TitleText>
+                    <SupportText>{data.user.email}</SupportText>
+                </ToolbarTitleItem>
+            </Toolbar>
+            <MainContent>
+                {/* TODO Use RouterTabs when working under subroutes */}
```

Comment:
This should work in next. @nsams can you investigate?

Reply:
that error can happen when you use <FormattedMessage as tab title or stack title?

Reply:
> that error can happen when you use <FormattedMessage as tab title or stack title?

I believe this is the reason for this error

Reply:
Yes it was.

Reply:
It's actually happening if you use `<FormattedMessage .../>` as tab label (even with the current version of Comet v5.0.0-canary-20230905121614).

We can use `label={intl.formatMessage({ id: "comet.userManagemant.basicData", defaultMessage: "Basic Data" })}` to support RouterTabs, what i would prefer, because every Tab should have an path to easily navigate to the data in the Tab (e.g. for sharing or Button switch to one of this Tabs).